testing/integration/test.py

```diff
@@ -0,0 +1,45 @@
+import requests
+#example api test
+
+import json
+import sys
+sys.path.append('../../taketwo-webapi')
+sys.path.append('../util')
```

Comment:
I'm new to python and dependencies so I'm not sure if this is optimal for being able to access adjacent files

Reply:
Cool! I'll update